client/src/components/Competitor/Competitor.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+interface CompetitorPropTypes {
```

Comment:
Looks good,

Could you move interfaces to the interfaces folder => constants/interfaces (Create the file and import and export it to/from the index.ts file in the interfaces folder. The naming convention has I in front of any interface you use, so you'd just change it to ICompetitorPropTypes and import it.

Reply:
I actually just went back and was working on implementing that naming convention before I saw your comment, however there are issues accessing react event types because the interface constants are outside of the client directory. Is there another way to more generically name React event types?

Reply:
I think you're okay to just leave the React import out of your ICompetitor.ts file. So it would look like this:

export default interface CompetitorPropTypes {
  competitor: string;
  competitorNum: string;
  competitorScore: number;
  handleScoreAdjust(e: React.MouseEvent, num: number, comp: string): void;
}

And then just include it in the index.tsx after for your export.

Reply:
If the react import is left out, then there is an error @ React.MouseEvent